api/v1alpha1/bundle_types.go

```diff
@@ -117,6 +117,7 @@ type BundleObject struct {
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name=Image,type=string,JSONPath=`.spec.source.image.ref`
+//+kubebuilder:printcolumn:name=Type,type=string,JSONPath=`.spec.source.type`
```

Comment:
Alternatively: we propagate this information from the status where we have the chance for more granularity, e.g. git -> git/tag, git/commit, etc.

Reply:
There might be a tradeoff here though where you're forcing provisioner implementations to also have to propagate this to the Bundle's status vs. referencing this type information directly from the spec, where we'll eventually require that this field gets set.